grafana/dev.dashboards.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: 1
```

Comment:
Is this only used for development?

Can you add a comment to this file describing what it's for, preferably with a reference to some source of truth for its format? (Guessing it's http://docs.grafana.org/v5.0/administration/provisioning/#dashboards)

Reply:
Yup, only for development. I've added a comment and referenced the Grafana docs.